packages/dd-trace/src/profiling/profilers/wall.js

```diff
@@ -120,6 +120,7 @@ class NativeWallProfiler {
       this._pprof.time.setContext(this._currentContext)
       this._lastSpan = undefined
       this._lastStartedSpans = undefined
+      this._webTags = undefined
```

Comment:
Can this be cleaned up after it's not longer needed so that the tags can be GCed?

Reply:
What do you mean? We'll keep updating this variable on every `async_hook.enter()` with the tags of some currently active span, so we don't hold on to any particular tags object for very long. We'll hold on longer to those that were active when we took a sample (once every 10ms) by transferring the reference into the `_currentContext`. That reference will go away next time we serialize the profile (once every minute.) We can't do better than that. In fact, we can – we can also release the tags as soon as the span ends; I do have that functionality in the other PR, #3755 where I added optimizations to release those as eagerly as possible, but Stephen considered it too hard to review at once so I started with this simplified one instead. I plan to add optimizations back in gradually.

One point of improvement here would be to reset the references when the profiler is stopped, but that won't make much difference.